firefox-ios/firefox-ios-tests/Tests/SyncIntegrationTests/Pipfile.lock

```diff
@@ -93,7 +93,7 @@
                 "sha256:fa3a0128b152627161ce47201262d3140edb5a5c3da88d73a1b790a959126956",
                 "sha256:fcc8eb6d5902bb1cf6dc4f187ee3ea80a1eba0a89aba40a5cb20a5087d961357"
             ],
-            "markers": "python_version >= '3.8'",
+            "markers": "platform_python_implementation != 'PyPy'",
```

Comment:
@AaronMT @jjSDET  is this change correct?
This PR is created automatically, I am not sure I have seen a version changed to a string like here...

Reply:
Confirmed on Slack that this is fine, we can merge this PR